test_runner/regress/test_timeline_delete.py

```diff
@@ -229,6 +229,8 @@ def test_delete_timeline_exercise_crash_safety_failpoints(
 
     ps_http.configure_failpoints((failpoint, "return"))
 
+    iterations = 20 if remote_storage_kind is RemoteStorageKind.REAL_S3 else 4
```

Comment:
Do we need to have sometimes faster timeouts, as in, will the quicker response ever be anything except flakyness?

It might be a slow failure if code is changed, but assuming that the tests work and the code works, then larger timeout => more stable test?

Reply:
Pasting from DMs:

> I'm still in the "fail fast" camp. This timeline test is parametrized which multiplies potential delay by the factor of 32. If we needlessly wait for 30 seconds instead of 10, this costs 20*32=640 seconds which is slightly above 10 minutes. To me it feels very important to keep test runs fast. They are not fast already, and timeouts make them even slower.
I wont die on that hill though, if you want me to set one big timeout I'll do that

Reply:
Yeah, well, this PR is improvement regardless so let's get it in.

Reply:
Discussed in DM's, ok to resolve